### ISSUE TITLE
Update to Paging 3.0

### ISSUE THREAD

**Issue by wangerekaharun** [wangerekaharun/AndroidPagingWithCoroutines#4]
Since Paging 3.0 already supports coroutines outside the box, time to update this project to use the latest library.

**Comment by ImL1s** [comment]
Paging 3.0 is alpha now, not suitable for production.

**Comment by wangerekaharun** [comment]
> Paging 3.0 is alpha now, not suitable for production.

Yes. the coroutines implementation is still on the master branch and there's a [Paging 3-0](https://github.com/wangerekaharun/AndroidPagingWithCoroutines/tree/paging-3.0) branch. Check the [ReadMe](https://github.com/wangerekaharun/AndroidPagingWithCoroutines/blob/master/README.md) for more details.
